README.md

```diff
@@ -29,59 +25,59 @@ sbot plugins.install ssb-query
 sbot plugins.install ssb-ws
 # restart sbot server (go back to previous tab and kill it)
 ```
-now clone and run patchbay.
-```
+
+now clone patchbay.
+
+```sh
 git clone https://github.com/ssbc/patchbay.git
 cd patchbay
 npm install
 npm run rebuild
```

Comment:
I think this `npm run rebuild` can be moved into the "Running the desktop app" section since it isn't needed for the lite client.